src/schemas/forms/script/parser.ts

```diff
@@ -61,11 +61,44 @@ const optionExportMatcher: SimpleVisitors<{ options: ObjectExpression | null }>
 };
 
 export function checkForChromium(objectExpression: ObjectExpression): boolean {
-  // Path to the property we're interested in
-  const path = ['scenarios', 'ui', 'options', 'browser', 'type'];
-  const value = getPropertyValueByPath(objectExpression, path);
+  const scenarios = getPropertyValueByPath(objectExpression, ['scenarios']);
+  if (!scenarios || scenarios.type !== 'ObjectExpression') {
+    return false;
+  }
+
+  for (const scenario of scenarios.properties) {
+    if (scenario.key.type === 'Identifier') {
+      const type = getPropertyValueByPath(scenario.value, ['options', 'browser', 'type']);
+      if (type === 'chromium') {
+        return true;
+      }
+    }
+  }
+
+  return false;
+}
+
+export function hasInvalidProperties(objectExpression: ObjectExpression): boolean {
+  const scenarios = getPropertyValueByPath(objectExpression, ['scenarios']);
+  if (!scenarios || scenarios.type !== 'ObjectExpression') {
+    return false;
+  }
+
+  const invalidProps = ['duration', 'vus', 'iterations'];
```

Comment:
If possible it might be better to be more granular with these. More specifically:

- `duration` must not be set, ever.
- `vus` must be either unset or `<= 1`
- `iterations` must be either unset or `<= 1`

Reply:
Specifically, what I'd prefer to avoid is rejecting a script with `{vus: 1, iterations: 1}`, as those are perfectly valid.

Reply:
Sure, let's tackle the validations in a more specific way.